CRM/Core/DAO/AllCoreTables.php

```diff
@@ -97,9 +97,12 @@ public static function tables() {
    * Get the declared token classes.
    * @return string[]
    *   [table_name => token class]
+   *
+   * @deprecated since 6.6 will be removed around 6.20.
    */
-  public static function tokenClasses() {
-    return array_column(self::getEntities(), 'token_class', 'name');
+  public static function tokenClasses(): array {
+    CRM_Core_Error::deprecatedFunctionWarning('use getClassesByProperty');
+    return \CRM_Core_DAO_AllCoreTables::getClassesByProperty('token_class');
```

Comment:
is there a reason not just to use `self::` here?

Reply:
I often prefer the long name for grep reasons